### PR TITLE
Add inconsistency api

### PR DESCRIPTION
**Pull request by mouthwatering**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 1]
```diff
@@ -74,16 +68,104 @@
...
+            "$ref": "./examples/LastDetect.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeseriesGroupId}/inconsistency/detect": {
+      "post": {
+        "summary": "Detect inconsistent time series from a group of similiar time series",
+        "description": " This operation helps detect the inconsistent series among a group series with similiar trend.",
+        "operationId": "DetectInconsistency",
```
Comment (kpajdzik) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json]: We recommend `{noun}_{verb}` pattern for operation IDs. It looks better in generated SDKs.

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 2]
```diff
@@ -74,16 +68,104 @@
...
+            "$ref": "./examples/DetectInconsistency.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeseriesGroupId}/inconsistency/query": {
+      "post": {
+        "summary": "Query time series properties from a group of similiar time series",
+        "description": " This operation helps query properties of one group, for example the inconsistent series among a group series with similiar trend.",
+        "operationId": "QueryTimeSeriesGroup",
```
Comment (kpajdzik) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json]: Same here

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 3]
```diff
@@ -108,7 +190,150 @@
...
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
```
Comment (kpajdzik) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json]: No 201 here?

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 4]
```diff
@@ -108,7 +190,150 @@
...
           "Detect anomaly status of the latest point in time series example": {
-            "$ref": "./examples/LastDetect.json"
+            "$ref": "./examples/QueryTimeSeriesGroup.json"
+          }
+        }
+      }
+    },
+    "/timeseriesgroups/{timeSeriesGroupId}": {
+      "get": {
+        "summary": "",
```
Comment (kpajdzik) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json]: Are all those summaries and descriptions intended to be empty?

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json [hunk 5]
```diff
@@ -140,6 +365,23 @@
         }
       }
     },
+    "Granularity": {
+      "type": "string",
+      "description": "Can only be one of yearly, monthly, weekly, daily, hourly or minutely. Granularity is used for verify whether input series is valid.",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "Granularity",
+        "modelAsString": false
```
Comment (kpajdzik) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json]: I know this was just moved but is there any reason why you don't want `modelAsString` set to true?

**Review by kpajdzik — CHANGES_REQUESTED** [review]
Waiting for the feedback response.

**Comment by KrisBash** [comment]
Does not appear to be a control plane API

**Comment by kpajdzik** [comment]
> Does not appear to be a control plane API

My bad!

**Comment by kpajdzik** [comment]
Closing as stale

**Pull request closed without merging**